daemon/openrazer_daemon/hardware/mouse.py

```diff
@@ -1665,6 +1665,22 @@ class RazerBasiliskV3ProWireless(RazerBasiliskV3ProWired):
     USB_PID = 0x00AB
 
 
+class RazerBasiliskV3Pro35KWired(RazerBasiliskV3ProWired):
+    """
+    Class for the Razer Basilisk V3 Pro 35K (Wired)
+    """
+
+    USB_PID = 0x00CC
```

Comment:
I'd rather you copy paste the Basilisk V3 Pro Wired config since it's two distinct device - even if they are still quite similar.

Also does your mouse not need `DPI_MAX = 35000` ?

Reply:
actually just fixed it up myself